components/hab/src/cli/hab/bldr.rs

```diff
@@ -61,6 +64,9 @@ pub enum Job {
 }
 
 /// Get the status of one or more job groups
+///
+/// NOTICE: Public Builder Build Functions are no longer supported. Please reach
+/// out to your account team if you were using these features.
 #[derive(ConfigOpt, StructOpt)]
 #[structopt(name = "status", no_version)]
 pub struct JobStatus {
```

Comment:
Not sure whether these arguments make sense any more for example if one runs `hab bldr job status`, a user is shown CLI parsing error (which is not same as the NOTICE above). When this is anyways not supported, don't think we need to deal with these options anymore.

Reply:
As discussed in daily and now noted in commit leaving these so that any scripting done leveraging these commands fail with the failure message instead of "missing option".

Reply:
I just tried locally something - and it works fine -  If we change 

```rust 
#[structopt(name = "status", no_version)]
``` 

to 

```rust
#[structopt(name = "demote", no_version, settings = &[AppSettings::ArgRequiredElseHelp])]
``` 

This avoids one round trip for user to get the `NOTICE: ...` 

I think it's a good idea to include that.